### ISSUE TITLE
pipenv installation dependency error

### ISSUE THREAD

**Issue by michaelhays** [django/channels_redis#70]
Running `pipenv install django channels channels_redis` in a new project:

``` bash
Adding channels_redis to Pipfile's [packages]…
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  You can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
Could not find a version that matches asgiref~=2.0.1,~=2.1
Tried: 0.8, 0.9, 0.9.1, 0.10.0, 0.11.0, 0.11.0, 0.11.0, 0.11.1, 0.11.1, 0.11.2, 0.11.2, 0.12.0, 0.12.0, 0.12.1, 0.12.1, 0.13.0, 0.13.0, 0.13.2, 0.13.2, 0.13.3, 0.14.0, 0.14.0, 1.0.0, 1.0.0, 1.0.1, 1.0.1, 1.1.0, 1.1.0, 1.1.1, 1.1.1, 1.1.2, 1.1.2, 2.0.0, 2.0.0, 2.0.1, 2.0.1, 2.1.0, 2.1.0
Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  You can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
Could not find a version that matches asgiref~=2.0.1,~=2.1
Tried: 0.8, 0.9, 0.9.1, 0.10.0, 0.11.0, 0.11.0, 0.11.0, 0.11.1, 0.11.1, 0.11.2, 0.11.2, 0.12.0, 0.12.0, 0.12.1, 0.12.1, 0.13.0, 0.13.0, 0.13.2, 0.13.2, 0.13.3, 0.14.0, 0.14.0, 1.0.0, 1.0.0, 1.0.1, 1.0.1, 1.1.0, 1.1.0, 1.1.1, 1.1.1, 1.1.2, 1.1.2, 2.0.0, 2.0.0, 2.0.1, 2.0.1, 2.1.0, 2.1.0
```

Got the same warning when running `pipenv install django channels asgi_redis`. Is `channels_redis` the new name for the `asgi_redis` package? Couldn't find anything about it, but assumed so since 2 was released 15 hours ago, and https://github.com/django/asgi_redis redirects to this repo. (**Update**: found [this blog post](http://www.aeracode.org/2018/02/02/channels-20/) which confirms that it was renamed)

I don't know much about dependency management, but is `asgiref~=2.0.1,~=2.1` the proper format for this?

**Comment by andrewgodwin** [comment]
Ah yes, it's because Channels needs one version while channels_redis declares another, older one. I'll fix it tonight - in the meantime, it's safe to skip the lock as the 2.1 version is forwards compatible.

**Comment by michaelhays** [comment]
Great! Thanks Andrew

**Comment by andrewgodwin** [comment]
This was fixed in 0e83eff1b63eeb5203b4abc55e6278fc411567ae and released as version `2.0.1`.
